internal/pkg/openapi/dynamic/provider.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2021 Terminus, Inc.
```

Comment:
move to internal/core/openapi/openapi-ng/routes/dynamic/register

Reply:
done